app/src/main/java/co/touchlab/kampkit/android/design/DimensionPresets.kt

```diff
@@ -0,0 +1,114 @@
+package co.touchlab.kampkit.android.design
+
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import co.touchlab.kampkit.android.design.Spacing.Surrounding.double_spacing
+import co.touchlab.kampkit.android.design.Spacing.Surrounding.half_spacing
+import co.touchlab.kampkit.android.design.Spacing.Surrounding.quadruple_spacing
+import co.touchlab.kampkit.android.design.Spacing.Surrounding.quarter_spacing
+import co.touchlab.kampkit.android.design.Spacing.Surrounding.quintuple_spacing
+import co.touchlab.kampkit.android.design.Spacing.Surrounding.septuple_spacing
+import co.touchlab.kampkit.android.design.Spacing.Surrounding.sextuple_spacing
+import co.touchlab.kampkit.android.design.Spacing.Surrounding.spacing
+import co.touchlab.kampkit.android.design.Spacing.Surrounding.triple_spacing
+
+object Spacing {
+    object Surrounding {
```

Comment:
Make this a `private object Spacings {` because we don't want people access this directly, instead want them accessing the rest.